src/valkey-benchmark.c

```diff
@@ -140,24 +146,27 @@ static struct config {
     int resp3; /* use RESP3 */
 } config;
 
+static struct placeholders {
```

Comment:
We have other placeholders too, e.g. `{tag}` so a comment can help understanding what this is.

```suggestion
/* Locations of the placeholders __rand_int__, __rand_1st__,
 * __rand_2nd, etc. within the RESP encoded command buffer. */
static struct placeholders {
```